src/common/interpolation.c

```diff
@@ -1395,17 +1366,15 @@ static void dt_interpolation_resample_plain(const struct dt_interpolation *itor,
   const int32_t out_stride_floats = out_stride / sizeof(float);
   int r;
 
-  debug_info("resampling %p (%dx%d@%dx%d scale %f) -> %p (%dx%d@%dx%d scale %f)\n", in, roi_in->width,
-             roi_in->height, roi_in->x, roi_in->y, roi_in->scale, out, roi_out->width, roi_out->height,
-             roi_out->x, roi_out->y, roi_out->scale);
+  dt_print_pipe(DT_DEBUG_PIPE | DT_DEBUG_VERBOSE, "resample_plain", NULL, itor->name, roi_in, roi_out, "\n");
+  dt_times_t start;
+  dt_get_times(&start);
```

Comment:
The start time should be set only if `DT_DEBUG_PERF` is set otherwise we are doing two system calls for nothing which is not really good performance wise. We have already this in the code in many places:

`if(darktable.unmuted & DT_DEBUG_PERF)  dt_get_times(&start);`

Another option is to introduce a macro for doing this.

There is multiple instances below.

Reply:
Good point... I'll fix.